mcrypt is not required, but it's recommended to increase file encryption
performance. We're adding it to info.xml just to simplify abuild (all app
dependencies can be read in info.xml).

--- a/apps/encryption/appinfo/info.xml
+++ b/apps/encryption/appinfo/info.xml
@@ -24,6 +24,7 @@
 	</types>
 	<dependencies>
 		<lib>openssl</lib>
+		<lib>mcrypt</lib>
 		<nextcloud min-version="12" max-version="12" />
 	</dependencies>
 	<settings>
